etl/steps/data/garden/lgbt_rights/2023-04-13/equaldex.py

```diff
@@ -0,0 +1,236 @@
+        "Varies by Region": "02. Varies by region",
+        "Ambiguous": "03. Ambiguous",
+        "Unequal": "04. Unequal",
+    }
+
+    id_blood = {
+        "Legal": "01. Legal",
+        "Varies by Region": "02. Varies by region",
+        "Ambiguous": "03. Ambiguous",
+        "Banned (3-month deferral)": "04. Banned (3-month deferral)",
+        "Banned (6-month deferral)": "05. Banned (6-month deferral)",
+        "Banned (1-year deferral)": "06. Banned (1-year deferral)",
+        "Banned (5-year deferral)": "07. Banned (5-year deferral)",
+        "Banned (indefinite deferral)": "08. Banned (indefinite deferral)",
+    }
+
+    id_censorship = {
+        "No censorship": "01. No censorship",
+        "Varies by Region": "02. Varies by region",
+        "Ambiguous": "03. Ambiguous",
+        "Other punishment": "04. Other punishment",
+        "Fine as punishment": "05. Fine as punishment",
+        "State-enforced": "06. State-enforced",
+        "Imprisonment as punishment": "07. Imprisonment as punishment",
+    }
+
+    id_conversion_therapy = {
+        "Banned": "01. Banned",
+        "Varies by Region": "02. Varies by region",
+        "Ambiguous": "03. Ambiguous",
+        "Not banned": "04. Not banned",
+    }
+
+    id_discrimination = {
+        "Illegal": "01. Illegal",
+        "Illegal in some contexts": "02. Illegal in some contexts",
+        "Varies by Region": "03. Varies by region",
+        "No protections": "04. No protections",
+    }
+
+    id_employment_discrimination = {
+        "Sexual orientation and gender identity": "01. Sexual orientation and gender identity",
+        "Sexual orientation only": "02. Sexual orientation only",
+        "Varies by Region": "03. Varies by region",
+        "Ambiguous": "04. Ambiguous",
+        "No protections": "05. No protections",
+    }
+
+    id_housing_discrimination = {
+        "Sexual orientation and gender identity": "01. Sexual orientation and gender identity",
+        "Sexual orientation only": "02. Sexual orientation only",
+        "Varies by Region": "03. Varies by region",
+        "Ambiguous": "04. Ambiguous",
+        "No protections": "05. No protections",
+    }
+
+    id_military = {
+        "Legal": "01. Legal",
+        "Don't Ask, Don't Tell": "02. Don't Ask, Don't Tell",
+        "Lesbians, gays, bisexuals permitted, transgender people banned": "03. Lesbians, gays, bisexuals permitted, transgender people banned",
+        "Ambiguous": "04. Ambiguous",
+        "Illegal": "05. Illegal",
+    }
+
+    id_non_binary_gender_recognition = {
+        "Recognized": "01. Recognized",
+        "Intersex only": "02. Intersex only",
+        "Varies by Region": "03. Varies by region",
+        "Ambiguous": "04. Ambiguous",
+        "Not legally recognized": "05. Not legally recognized",
+    }
+
+    # Match id variables with the new ids in new columns
+    df["homosexuality"] = df["value_formatted_homosexuality"].map(id_homosexuality)
```

Comment:
Just so you know, we have a `datautils.dataframes.map_series` function which has some improvements with respect to the raw `map` pandas method.